Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM openjdk:8
-COPY ./target/stuff-doer-1.0-SNAPSHOT-shaded.jar /usr/local/stuff-doer/stuff-doer-1.0-SNAPSHOT-shaded.jar
+COPY ./target/scala-2.11/stuff-doer-assembly-1.0-SNAPSHOT.jar /usr/local/stuff-doer/stuff-doer-1.0-SNAPSHOT-shaded.jar
```

Comment:
Maybe it would be better also to change the name of the file to where it is copied.
And then update the `CMD` line also with the new file name.

Reply:
will do